app/controllers/listings_controller.rb

```diff
@@ -873,7 +875,7 @@ def numeric_field_ids(custom_fields)
   end
 
   def normalize_price_params(listing_params)
-    currency = listing_params[:currency]
+    currency = @current_community.currency
```

Comment:
You could make this method a bit more pure by reading the currency from `listing_params[:currency]`